.github/workflows/semconvgen.yml

```diff
@@ -24,7 +24,9 @@ jobs:
         pip install -U setuptools wheel
         pip install -r dev-requirements.txt
     - name: Check formatting (black)
-      run: black --check --diff .
+      run: |
+        pip install -U black
```

Comment:
Please don't change this. black is (should be!) already listed in dev-requirements.txt. Re-installing it here would ignore any version requirements in dev-requirements.txt